addons/binding/org.openhab.binding.plugwise/src/main/java/org/openhab/binding/plugwise/handler/AbstractPlugwiseThingHandler.java

```diff
@@ -0,0 +1,291 @@
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.plugwise.handler;
+
+import static org.eclipse.smarthome.core.thing.ThingStatus.*;
+import static org.eclipse.smarthome.core.thing.ThingStatusDetail.BRIDGE_OFFLINE;
+import static org.openhab.binding.plugwise.PlugwiseBindingConstants.CHANNEL_LAST_SEEN;
+
+import java.math.BigDecimal;
+import java.time.Duration;
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Map;
+
+import org.eclipse.smarthome.config.core.Configuration;
+import org.eclipse.smarthome.core.thing.ChannelUID;
+import org.eclipse.smarthome.core.thing.Thing;
+import org.eclipse.smarthome.core.thing.ThingStatus;
+import org.eclipse.smarthome.core.thing.ThingStatusDetail;
+import org.eclipse.smarthome.core.thing.ThingStatusInfo;
+import org.eclipse.smarthome.core.thing.binding.BaseThingHandler;
+import org.eclipse.smarthome.core.types.Command;
+import org.openhab.binding.plugwise.PlugwiseBindingConstants;
+import org.openhab.binding.plugwise.internal.PlugwiseCommunicationHandler.MessagePriority;
+import org.openhab.binding.plugwise.internal.PlugwiseDeviceTask;
+import org.openhab.binding.plugwise.internal.PlugwiseUtils;
+import org.openhab.binding.plugwise.internal.listener.PlugwiseMessageListener;
+import org.openhab.binding.plugwise.internal.protocol.InformationRequestMessage;
+import org.openhab.binding.plugwise.internal.protocol.InformationResponseMessage;
+import org.openhab.binding.plugwise.internal.protocol.Message;
+import org.openhab.binding.plugwise.internal.protocol.PingRequestMessage;
+import org.openhab.binding.plugwise.internal.protocol.field.DeviceType;
+import org.openhab.binding.plugwise.internal.protocol.field.MACAddress;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The {@link AbstractPlugwiseThingHandler} handles common Plugwise device channel updates and commands.
+ *
+ * @author Wouter Born - Initial contribution
+ */
+public abstract class AbstractPlugwiseThingHandler extends BaseThingHandler implements PlugwiseMessageListener {
+
+    private static final int DEFAULT_UPDATE_INTERVAL = 60;
+    private static final int MESSAGE_TIMEOUT = 15;
+    private static final int MAX_UNANSWERED_PINGS = 2;
+
+    private final PlugwiseDeviceTask onlineStateUpdateTask = new PlugwiseDeviceTask("Online state update", scheduler) {
+        @Override
+        public int getConfiguredInterval() {
+            return MESSAGE_TIMEOUT;
+        }
+
+        @Override
+        public void runTask() {
+            updateOnlineState();
+        }
+
+        @Override
+        public boolean shouldBeScheduled() {
+            return shouldOnlineTaskBeScheduled();
+        }
+
+        @Override
+        public void start() {
+            onlineStateUpdateTask.update(getDeviceType(), getMACAddress());
```

Comment:
The line has now been removed since it is actually redundant.

According to the call hierarchy and some testing `.start()` is always called via the `.updateTask(..)` method (similar to all other recurring tasks). This method always makes sure the properties are updated before starting the task: 

```
                task.update(getDeviceType(), getMACAddress());
                task.start();
```